code/modules/mob/living/carbon/human/species/station/traits_vr/negative.dm

```diff
@@ -133,3 +133,18 @@
 /datum/trait/colorblind/para_taj/apply(var/datum/species/S,var/mob/living/carbon/human/H)
 	..(S,H)
 	H.add_modifier(/datum/modifier/trait/colorblind_taj)
+
+/datum/trait/coldprone
+	name = "Cold prone"
+	desc = "You are much more prone to colder temperatures than other species, and become uncomfortable much more easily in cold environments."
+	cost = -2
+	var_changes = list("cold_level_1" = 280, "cold_level_2" = 220, "cold_level_3" = 140, "breath_cold_level_1" = 260, "breath_cold_level_2" = 240, "breath_cold_level_3" = 120, "cold_discomfort_level" = 280)
```

Comment:
How would simultaneously taking one of those new traits and hot-blooded/cold-blooded affect the final comfort level? Since both apply changes to it, they should either be mutually exclusive or new traits shouldn't change comfort specifically, I think.

Reply:
Good point that would probably fuck it. Welp, time to make even more commits.